docs/source/docs/level-up/intermediate_skills/07_data_merge.rst

```diff
@@ -2,4 +2,81 @@
 Level 7: Merge Two Heterogeneous Datasets
 =========================================
 
-TBD
+
+In the latest deep learning trends, training foundation models with larger datasets has become
+increasingly popular. To achieve this, it is crucial to collect and prepare massive datasets for deep
+learning model development. Collecting and labeling large datasets can be challenging, so
+consolidating scattered datasets into a unified one is important. For instance, `Florence <https://arxiv.org/pdf/2111.11432.pdf>`_
+created the FLOD-9M massive dataset by combining MS-COCO, LVIS, OpenImages, and Object365 datasets
+to use for training.
+
+In this tutorial, we provide the simple example for merging two datasets and the detailed description
+for merge operation is given by :ref:`here <Merge>`.
+The more advanced Python example with the label mapping between datasets is given
+`here <https://github.com/openvinotoolkit/datumaro/blob/develop/notebooks/01_merge_multiple_datasets_for_classification.ipynb>`_.
```

Comment:
![image](https://user-images.githubusercontent.com/26541465/233248698-053e8696-32a6-4cae-96c9-27f74eb324f7.png)

Please use relative link as above. This might be working after https://github.com/openvinotoolkit/datumaro/pull/958 since `releases/1.2.0` doesn't enroll the notebook examples in the toctree yet.

Reply:
I fixed.